TipCatDotNet.Api/Data/Models/Owner.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace TipCatDotNet.Api.Data.Models
+{
+    [Table("accounts")]
+    public class Owner
```

Comment:
Use one term for program classes and database entities. This is much safer and clearer.